src/booleans.js

```diff
@@ -1,61 +1,86 @@
+/* eslint-disable no-unreachable */
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return !(a || b);
 };
 
 function one(a, b) {
-  // your code here
+  if ((a === true && b === false) || (a === false && b === true)) {
+    return true;
+  } else return false;
```

Comment:
The solution here works, but is very verbose when it doesnt need to be. Remember `a` & `b` are already boolean values, so they dont need to have the comparison to determine if they are truthy or falsy, it could be simplified to 
```
if ((a && !b) || (!a && b)) {
  return true;
} else {
  return false;
}
```
This could be simplified further:

```
return (a && !b) || (!a && b);
```